server/service/osquery_utils/queries.go

```diff
@@ -167,7 +167,7 @@ var hostDetailQueries = map[string]DetailQuery{
 		Query: `
 	SELECT
 		os.name,
-		os.codename as display_version
+		os.version as display_version
```

Comment:
One last nit, could you remove the comment in line 166? (it's outdated, from when the query was using `registry` table)